docs/using.rst

```diff
@@ -491,7 +497,7 @@ the Nginx plugin is ready.
    emerge -av app-portage/layman
    layman -S
    layman -a mrueg
-   emerge -av app-crypt/letsencrypt-nginx
+   emerge -av app-crypt/certbot-nginx
```

Comment:
I think everything about using layman here should be deleted. We have very little information about it and I'm not sure if a `certbot-nginx` package exists. We removed all info about it from the website before it launched.